### DISCUSSION TITLE
Adding Fuel type

### DISCUSSION THREAD

**Discussion by Dhal1a** [firemodels/fds#11406] [Q&A]
Hi, I am trying to setup polyurethane in my file, 

I used to use FDS over 10 years ago and I admit I am very much out of touch now, so would greatly appreciate help 
ERROR: Simple chemistry fuel, POLYURETHANE, not defined on REAC or SPEC.

[WP1_1_SLOW.txt](https://github.com/firemodels/fds/files/10546773/WP1_1_SLOW.txt)

**Comment by mcgratta** [comment]
You need to provide a definition of POLYURETHENE, either on the REAC line or SPEC line. For example, you have to define the C, H, O
